docs/trussels.md

```diff
@@ -362,29 +351,15 @@ They are all great and approachable.
 
 The Truss leadership team includes the founding team and:
 
-* **Amy Meng** Delivery Management Practice Lead.
-* **Carla Silvey** Product Management Practice Lead.
-* **Carmen Bocanegra** Research and Design Practice Lead.
-* **Elizabeth Eady** (Temporary) Infrastructure Engineering Practice Lead.
-* **Hannah Kane** Head of Delivery (oversees Client Success Managers and Delivery Managers)
-* **Nick Twyman** Application Engineering Practice Lead.
-* **Nora Alves** (Interim) Head of People.
-
-## Resource: Departments and Practice Areas
-Truss has three departments led by the founders:
-
-* BizOps (business operations), led by Jen Leech (COO)
-* Key Practices (the product/service we sell), led by Mark Ferlatte (CTO)
-* Sales (commercial and government), led by Everett Harper (CEO)
+* **Amy Meng** Associate Director of Delivery Management
+* **Carla Silvey** Director of Product Management
+* **Carmen Bocanegra** Director of Research and Design 
+* **Elizabeth Eady** (Temporary) Director of Infrastructure Engineering
+* **Hannah Kane** VP of Delivery (oversees Client Success Managers and Delivery Managers)
```

Comment:
Please correct Hannah's title. In general, Paylocity is the source of truth here.

Reply:
Paylocity lists Hannah's title as "Vice President, Delivery" -- should it be something different?

Reply:
LOL. OK, hoisted on my own Paylocity petard. Apologies for assuming.